packages/@react-stately/grid/src/GridCollection.ts

```diff
@@ -150,4 +150,9 @@ export class GridCollection<T> implements IGridCollection<T> {
   getItem(key: Key) {
     return this.keyMap.get(key);
   }
+
+  getAt(idx: number) {
```

Comment:
Can we call this just `at` to match the new [Array#at](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at) function being added to JavaScript?